.github/workflows/syncer-image.yaml

```diff
@@ -19,6 +19,6 @@ jobs:
         with:
           go-version: v1.17
 
-      # Build and push a syncer image, tagged with the commit SHA.
+      # Build and push a syncer image, tagged with the commit SHA and the branch name.
       - uses: imjasonh/setup-ko@v0.4
-      - run: ko publish ./cmd/syncer -t $(git rev-parse --short "$GITHUB_SHA")
+      - run: ko publish ./cmd/syncer -t $(git rev-parse --short "$GITHUB_SHA"),${{ github.ref_name }}
```

Comment:
how will the syncer image be tagged? Does this mean here that there won't be a git tag on the syncer image?

Reply:
it should have two tags:
- git rev-parse --short "$GITHUB_SHA", which is the short version of the commit
- github.ref_name, the branch name
Example: https://github.com/fgiloux/kcp/pkgs/container/kcp%2Fsyncer-c2e3073d5026a8f7f2c47a50c16bdbec
It has b952d3d and release-0.42

Reply:
how can we get a tagged one too?

Reply:
Tags would need to be added to the event list:
~~~
on:
  push:
    branches:
    - main
    - 'release-*'
    tags:
    - 'v*'
~~~

I can add it to this PR or open a separate one if you prefer. Would you want the action to be triggered on every tag or on tags having a specific pattern v* above?

Reply:
`v*`

Reply:
Just add it here I would say.

Reply:
I have pushed the change here.